glossary.md

```diff
@@ -476,6 +486,9 @@ Cross-references:
 **<a name="enterprise-architecture-board"></a>[Enterprise Architecture Board](#enterprise-architecture-board)** (EAB) \
 _No definition provided._
 
+**<a name="entrance-on-duty"></a>[Entrance on duty](#entrance-on-duty)** (EOD) \
+The date on which a person completes the necessary paperwork and is sworn in as an employee
```

Comment:
is the other use of EOD common enough that it's not worth noting it's overloaded here?

Reply:
yeah  - End-of-day